web/js/date/model.js

```diff
@@ -35,8 +35,14 @@ export function dateModel(config, spec) {
   };
 
   self.clamp = function (date) {
-    if (date > util.today()) {
-      date = util.today();
+    if (self.maxZoom > 3) {
```

Comment:
Daily layers need the newest date clamped to 00:00 time otherwise the daily layer will be shown as unavailable because their endDate is looking for today at 00:00. 

If a subdaily layer is active, then the maxZoom is 4 and the time input + minute zoom level is exposed which will allow you to move the time values as needed. 

Also, when the app is loaded, the hour and minute input selectors won't be available, so you'll want the newest date (today) to start with today with 00:00 time since you can't change this value except in the URL.